Singapore/src/main/java/com/infoshareacademy/service/LoginUser.java

```diff
@@ -0,0 +1,58 @@
+package com.infoshareacademy.service;
+
+import com.infoshareacademy.model.User;
+
+import java.util.Scanner;
+
+public class LoginUser extends User {
+    Scanner input = new Scanner(System.in);
+
+    // Set  user: login, first name, last name and password
+
+    String login = "1";
+    String password = "1";
+
+
+    // Ask user to enter: login and password
+
+//    @Override
+//    public User createUser() {
+//        return super.createUser();
+//    }
+
+    public void doLogin() {
```

Comment:
```suggestion
    public void login() {
```
Ta metoda powinna się znaleźć w klasie UserService ponieważ wykonuje operacje na userze.